Gemfile

```diff
@@ -240,3 +240,5 @@ group :development, :test do
   gem 'spring'
   gem 'spring-watcher-listen'
 end
+
+gem "dockerfile-rails", ">= 1.5", :group => :development
```

Comment:
Do we need this in the dependencies, our current Dockerfile is hand-rolled anyway. 

CC: @sojan-official let me know your thoughts, I don't have much context on this

Reply:
Railway resolved the issue with the template.  I deployed it without any issue.